L1Trigger/L1THGCal/src/concentrator/HGCalConcentratorSuperTriggerCellImpl.cc

```diff
@@ -99,16 +99,19 @@ void HGCalConcentratorSuperTriggerCellImpl::assignSuperTriggerCellEnergyAndPosit
                                                                                     const SuperTriggerCell& stc) const {
   //Compress and recalibrate STC energy
   uint32_t compressed_value = getCompressedSTCEnergy(stc);
-
+  SubDetectorType subdet = hgcal_silicon_CEE;
   int thickness = 0;
   if (triggerTools_.isSilicon(c.detId())) {
     thickness = triggerTools_.thicknessIndex(c.detId(), true);
+    if (triggerTools_.isEm(c.detId())) subdet = hgcal_silicon_CEE;
+    else if (!triggerTools_.isEm(c.detId())) subdet = hgcal_silicon_CEH;
   } else if (triggerTools_.isScintillator(c.detId())) {
     thickness = 3;
```

Comment:
I probably missed this hardcoded value in a previous PR. Should be replaced by `HGCalTriggerTools::kScintillatorPseudoThicknessIndex_`